website/source/docs/secrets/mysql/index.html.md

```diff
@@ -248,13 +248,21 @@ the default on versions prior to that.
       <li>
         <span class="param">sql</span>
         <span class="param-flags">required</span>
-        The SQL statements executed to create and configure the role. Must be
-        a semicolon-separated string, a base64-encoded semicolon-separated
+        The SQL statements executed to create and configure a user. Must be a
+        semicolon-separated string, a base64-encoded semicolon-separated
         string, a serialized JSON string array, or a base64-encoded serialized
         JSON string array.  The '{{name}}' and '{{password}}' values will be
         substituted.
       </li>
       <li>
+        <span class="param">revocation_sql</span>
```

Comment:
We can change this later if we want; I think due to the similarity with the `sql` parameter it should be pretty obvious.